### ISSUE TITLE
noEnchants/noGems not working

### ISSUE THREAD

**Issue by Elehiggle** [Tercioo/Open-Raid-Library#13]
Says that I don't have any enchants or gems, while being fully enchanted and gemmed.

**Comment by Tercioo** [comment]
makes sense, as the lib is getting the data for Dragonflight due to the 
game version be 10.0.0 now
ill see if this can be reverted until DF launch

------ Original Message ------
From: "Elehiggle" ***@***.***>
To: "Tercioo/Open-Raid-Library" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: 28-Oct-22 11:54:09 AM
Subject: [Tercioo/Open-Raid-Library] noEnchants/noGems not working 
(Issue #13)
